doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -73,6 +74,41 @@ enhancements of NGSIv2 subscriptions (e.g. filtering) with NGSIv1 legacy notific
 
 [Top](#top)
 
+## Datetime support
```

Comment:
@iariasleon pay attention to this.

In the past, we were discussing that the particular implementation supports of ISO8601 was not clear and that situation was precluding the implementation of test. This documentation should be precise enough (so the impediment to develop new tests disappear) but if you find it not clear, please raise a comment.